app/src/main/res/raw/data.json

```diff
@@ -7620,31 +7620,34 @@
 	[
     {"EN":{
         "word": "thousand",
```

Comment:
I think, all former "numbers" are  good to add to "maths and measurement"

Reply:
+1